src/utils/ohdf/calculations.ts

```diff
@@ -0,0 +1,108 @@
+// utils/calculations.ts
+
+import _ from 'lodash'
+import flat from 'flat'
+import {ContextualizedEvaluation, ContextualizedProfile} from 'inspecjs'
+import {calculateCompliance, extractStatusCounts, renameStatusName, severityTargetsObject} from '../threshold'
```

Comment:
Does it make sense to add these to this file instead of having them in the threshold util? It seems we want to have overall utils organized based on the functionality rather than where they are used and it makes sense to me to have the calculate stuff together.

Reply:
I didn't write that code, but will be rewriting it as part of the thresholds PR.

Short answer:

No given I think this calculations parent will have more kids soon ...

Longer Answer:

Given the complexity or hopefully not so complex after, we will likely have a construction like:

```shell
./src/utils/ohdf/calculations/thresholds.ts
...../same/summary.ts
.../same/other thing we have to count.ts
```

I am finding, as with the outputGenerators, which I make make this change, that scoping things to their utilityTYpe is really the way to go.

Reply:
```suggestion
import {calculateCompliance, extractStatusCounts, renameStatusName, severityTargetsObject} from '../threshold'
```

Create an issue we can attach to the PR for 'fixing summary'